superset-frontend/src/explore/exploreUtils/getChartKey.test.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { getChartKey } from '.';
+
+test('should return 0 when called with complete object', () => {
```

Comment:
Not sure what the title means here... if the assertion is that it returns 100, why is the test title saying 0?

And the "when called with a complete object" title begs the question: are there times it may NOT be called in that condition that we should assert against?

Reply:
This file had other tests with "skip" to show some very dangerous behaviors that this function has.
But the guys warned me that it was better to remove comments and tests with skip because the correct solution would be to do another PR fixing these problems.
When I removed the other tests this name and description really lost all context.